### PR TITLE
Checkstyle: require block braces, standardize indentation.

### PR DESCRIPTION
**Pull request by bretthoerner**
#298 

This is a TODO for me. I need to knockout the swath of changes when there aren't as many PRs waiting.

### CODE REVIEW

**Comment by bretthoerner** [comment]
Inspiration: https://github.com/checkstyle/checkstyle/blob/checkstyle-7.5/src/main/resources/google_checks.xml

**Comment by bretthoerner** [comment]
Also need to disable check on Java 7 Travis run. https://docs.travis-ci.com/user/customizing-the-build

**Comment by bretthoerner** [comment]
:heart: IntelliJ. Did all this automatically.

File: pom.xml [hunk 1]
```diff
@@ -364,6 +364,14 @@
                         <configLocation>src/checkstyle/checkstyle.xml</configLocation>
                         <consoleOutput>true</consoleOutput>
                     </configuration>
+                    <dependencies>
+                        <dependency>
+                            <groupId>com.puppycrawl.tools</groupId>
```
Comment (tkaemming) [inline pom.xml]: 🐶

**Pull request merged**